dcm4ceph-tool/src/main/java/org/open_ortho/dcm4ceph/tool/ceph2dicomdir/Ceph2Dicom.java

```diff
@@ -67,62 +67,73 @@ public static void main(String[] args) {
 				.desc("Name of cephalogram x-ray image file.")
 				.build();
 		inputfile.setArgs(1);
-
 		Option fiducialfile = Option.builder("f")
 				.longOpt("fiducialfile")
 				.argName("file")
 				.optionalArg(true)
 				.numberOfArgs(1)
 				.desc("Name of fiducials punched on the x-ray file.")
 				.build();
-
 		Option outputdir = Option.builder("o")
 				.longOpt("outputdir")
 				.argName("directory")
 				.hasArg()
 				.desc("directory where to save DICOM into. Defaults to this one.")
 				.build();
-
 		Options options = new Options();
 		options.addOption("B", "boltonset", false,
 				"create DICOMDIR of a Bolton Set with PA, Lateral, and Fiducial. Requires specifying --file twice: PA, Lateral and --fiducialfile.");
 		options.addOption(inputfile);
 		options.addOption(fiducialfile);
 
+		CommandLine line;
+		try {
+			line = parser.parse(options, args);
+		} catch (ParseException exp) {
+			exp.printStackTrace();
+			Log.err("Incorrect Arguments.");
+			HelpFormatter formatter = new HelpFormatter();
+			formatter.printHelp("ceph2dicom", options);
+			System.exit(1);
+			return;
+		}
+
 		try {
-			CommandLine line = parser.parse(options, args);
 			if (line.hasOption("boltonset")) {
-				System.out.println("Converting PA, Lateral and Fiducial Set to DICOMDIR.");
+				// boltonset mode of operation
+				Log.info("Converting PA, Lateral and Fiducial Set to DICOMDIR.");
 				File cephfile1 = new File(line.getOptionValues(inputfile)[0]);
 				File cephfile2 = new File(line.getOptionValues(inputfile)[1]);
 				File fidfile = new File(line.getOptionValue(inputfile));
 				BBCephalogramSet cephSet = new BBCephalogramSet(cephfile1, cephfile2,
 						fidfile);
-
 				cephSet.writeDicomdir(new File(cephfile1.getParent() + File.separator
 						+ "BBcephset"));
-
-			} else {
-				System.out.println("Converting single TIF file to DCM.");
-				String outputDirectory = null;
-				if (line.hasOption(outputdir))
-					outputDirectory = line.getOptionValue(outputdir);
-
-				Cephalogram ceph = new Cephalogram(new File(line.getOptionValue(inputfile)));
-				if (outputDirectory != null)
-					ceph.writeDCM(outputDirectory, null);
-				else
-					ceph.writeDCM();
-
+				return;
 			}
-
-			// printDicomElements(FileUtils.getDCMFile(cephfile));
-
-		} catch (ParseException exp) {
-			System.err.println("ERR: Incorrect Arguments.");
-			System.err.println(exp);
-			HelpFormatter formatter = new HelpFormatter();
-			formatter.printHelp("ceph2dicom", options);
+			// single file mode of operation
+			Log.info("Converting single TIF file to DCM.");
```

Comment:
It's not converting TIF files yet. Just remove the TIF to read "converting single file to DCM".

Reply:
Done